### ISSUE TITLE
There is no way to dynamically change `forceRender` property

### ISSUE THREAD

**Issue by KryDos** [palerdot/vue-speedometer#4]
hi @palerdot, 
Thanks for the lib, it's great. 

My current issue is that if `forceRender` property is initially true then I cannot change it to false dynamically. 

My flow is this:

* speedometer's forceRender = true
* make http request to the server to get max value for the speedometer 
* set that max value and set forceRender to `false` because I don't need to update the whole speedometer anymore. 
* speedometer does not react to the change for forceRender = false

I believe it happens because of [THIS LINE](https://github.com/palerdot/vue-speedometer/blob/11a88595c80e6ede5951ebe77e5fa607f1c1a184/src/index.js#L40).

```
      this.force_render = newValue || oldValue
```

This code means that if newValue is false then we automatically apply oldValue which is true. So there is no way to set forceRender to be false if it is already true. 

Could you please let me know if this piece of code is there intentionally or we can just apply value from `newValue` variable with no issues?

**Comment by palerdot** [comment]
@KryDos Thanks for raising the issue. You are right. For boolean values, it is enough to just take the new values. so just 
```
this.force_render = newValue
```
should be ok. I will try to add tests before publishing. Even without that, I will just try to release this as early as possible as `v1.1.0`. I will post updates on this issue.

**Comment by palerdot** [comment]
@KryDos This issue is fixed and release as `v1.1.0`. A simple simulation of your scenario is in this codesandbox - https://codesandbox.io/s/vue-template-t5j5n. Please reopen this issue if you still face problems.

**Comment by KryDos** [comment]
Brilliant, @palerdot. Thanks a lot!
